### PR TITLE
[statping] Rebase statping on common chart

### PR DESCRIPTION
**Pull request by angelnu**
**Description of the change**

Rebase statping on common chart

**Benefits**

- More configuration options from the common chart.
- Less maintenance effort

**Possible drawbacks**

- Users need to adapt their configuration when moving to this mayor upgrade.

**Applicable issues**

- fixes NA

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] (optional) Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [x] (optional) Variables are documented in the README.md

### CODE REVIEW

File: charts/statping/README_CHANGELOG.md.gotmpl [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+### [2.0.0]
+
+#### Added
+
+- N/A
+
+#### Changed
+
+- Redone based on common chart
```
Comment (bjw-s) [inline charts/statping/README_CHANGELOG.md.gotmpl:20]:
```suggestion
- Migrated to common library chart
```

File: charts/statping/templates/common.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  {{- $_ := set .Values.env "DB_USER" .Values.postgresql.postgresqlUsername -}}
+{{- end -}}
+{{/* if TTRSS_DB_USER is not set use internal postgres chart */}}
+{{- if eq .Values.env.DB_PASS "" -}}
+  {{- $_ := set .Values.env "DB_PASS" .Values.postgresql.postgresqlPassword -}}
+{{- end -}}
+{{/* if TTRSS_DB_USER is not set use internal postgres chart */}}
+{{- if eq .Values.env.DB_DATABASE "" -}}
+  {{- $_ := set .Values.env "DB_DATABASE" .Values.postgresql.postgresqlDatabase -}}
+{{- end -}}
```
Comment (bjw-s) [inline charts/statping/templates/common.yaml:22]: I'm pretty sure there was some copy/pasting going on here from the TTRSS chart 😉

File: charts/statping/values.yaml [hunk 3]
```diff
@@ -1,171 +1,83 @@
...
+  # -- dashboard admin user (needed to edit)
+  ADMIN_USER: admin
+  # -- dashboard admin password (needed to edit)
+  ADMIN_PASSWORD: admin
+  # -- dashboard admin email
+  ADMIN_EMAIL: info@example.com
+  # -- Use CDN for static context from third-parties
+  USE_CDN: false
+  # -- Disable logs from appearing and writing to disk
+  DISABLE_LOGS: false
```
Comment (bjw-s) [inline charts/statping/values.yaml:50]: Please replace the `# --` with `##` (also in the rest of the file) to keep the comments consistent with the other charts

File: charts/statping/values.yaml [hunk 4]
```diff
@@ -1,171 +1,83 @@
...
-
-tolerations: []
-
-affinity: {}
-
-podAnnotations: {}
+# -- Bitnami postgres chart. For more options see https://github.com/bitnami/charts/tree/master/bitnami/postgresql
+# @default -- see bellow
+postgresql:
+  # -- By default uses an internal postgress. Dissable if you use your own Postgres.
```
Comment (bjw-s) [inline charts/statping/values.yaml:72]:
```suggestion
  ## By default uses an internal postgress. Disable if you use your own Postgres.
```

**Review by bjw-s — CHANGES_REQUESTED** [review]
Hi, thanks for taking the time to submit this PR! Some small changes requested, but looks great for the most part!

File: charts/statping/README_CHANGELOG.md.gotmpl (hunk 1 above)
Comment (Ornias1993) [reply charts/statping/README_CHANGELOG.md.gotmpl:20]: Might also be worth while noting that service.port has moved to service.port.port (as thats one of the bigger breaking changes)

File: charts/statping/values.yaml (hunk 3 above)
Comment (onedr0p) [reply charts/statping/values.yaml:50]: I think the comment format are for helmdocs

Comment (angelnu) [reply charts/statping/values.yaml:50]: I tried but then the descriptions were not recognized. Is there a new version/task to use for `##` to work with helm-docs?

Comment (onedr0p) [reply charts/statping/values.yaml:50]: @angelnu I think you have it right, it's something that should be updated across our charts so helm docs can pick up the descriptions

File: charts/statping/templates/common.yaml (hunk 2 above)
Comment (angelnu) [reply charts/statping/templates/common.yaml]: Indeed :-D

File: charts/statping/README_CHANGELOG.md.gotmpl (hunk 1 above)
Comment (angelnu) [reply charts/statping/README_CHANGELOG.md.gotmpl]: Added - thanks for the feedback

**Review by bjw-s — APPROVED** [review]

**Pull request merged**